src/components/Project/FundingCycleSection.tsx

```diff
@@ -55,23 +51,14 @@ export default function FundingCycleSection({
         {reconfigureButton}
       </div>
 
-      <Space style={{ fontSize: '.8rem', marginBottom: 12 }} size="middle">
+      <Space style={{ fontSize: '.8rem', marginBottom: 20 }} size="middle">
```

Comment:
**One small nit-pick:** I think we should make this `size="large"` to match the other tab instances.
(Safe has more space):
<img width="162" alt="Screen Shot 2022-10-31 at 10 55 35 am" src="https://user-images.githubusercontent.com/96150256/198911441-b91bdcfa-1de8-490f-9ad5-b98ba1a0716f.png">
<img width="185" alt="Screen Shot 2022-10-31 at 10 55 31 am" src="https://user-images.githubusercontent.com/96150256/198911443-82f9d7fe-d2ee-49fd-a84e-7ae2435dbf0d.png">

Reply:
yes, agreed! looks much better, pushing!

Reply:
Sweet!